allocation.md

```diff
@@ -307,6 +307,18 @@ $ curl http://localhost:9090/model/allocation \
 }
 ```
 
+### Notes on Asset Cost Distribution Methodology:
```

Comment:
In general, it's fine to leave out "Notes on" in documentation and just title a section with substance. "Asset Cost Distribution" would be a fine title. I'd also float the idea of calling this "Allocation of Asset Costs" or similar, to make the target of the distribution clear; as in, "Where are the assets costs being distributed? They are being allocated to allocations."